pkg/cli/cmd/directory/test.go

```diff
@@ -2,49 +2,39 @@ package directory
 
 import (
 	"context"
+	"encoding/csv"
 	"encoding/json"
 	"errors"
 	"fmt"
 	"os"
 	"strings"
-	"sync/atomic"
 	"time"
 
 	dsr3 "github.com/aserto-dev/go-directory/aserto/directory/reader/v3"
 	"github.com/aserto-dev/topaz/pkg/cli/cc"
 	dsc "github.com/aserto-dev/topaz/pkg/cli/clients/directory"
+	"github.com/aserto-dev/topaz/pkg/cli/cmd/common"
 
-	"github.com/fatih/color"
-	"github.com/samber/lo"
 	"google.golang.org/protobuf/encoding/protojson"
-	"google.golang.org/protobuf/proto"
 	"google.golang.org/protobuf/types/known/structpb"
 )
 
-const (
-	check           string = "check"
-	checkRelation   string = "check_relation"
-	checkPermission string = "check_permission"
-	expected        string = "expected"
-	passed          string = "PASS"
-	failed          string = "FAIL"
-	errored         string = "ERR "
-)
-
 type TestCmd struct {
 	Exec     TestExecCmd     `cmd:"" help:"execute assertions"`
 	Template TestTemplateCmd `cmd:"" help:"output assertions template"`
 }
 
 type TestExecCmd struct {
 	File    string `arg:""  default:"assertions.json" help:"filepath to assertions file"`
```

Comment:
We can do it in a separate PR but I think it would be nice to allow multiple files to be specified. 
```go
Files  []string `arg:"" default:"assertions.json" sep:"none" help:"path to assertions file"`
```
That would allow users to use shell expansions and run a set of test files in a single call:
```shell
topaz ds test exec test/*.json
```

And also do things like:
```shell
cat tests.txt | xargs topaz ds test exec
```